.claude/commands/dev/all-tools.md

```
@@ -0,0 +1,31 @@
+
+Example format:
+```typescript
+• functionName(parameters: Type): ReturnType - Purpose of the tool
+
+• anotherFunction(params: ParamType): ResultType - What this tool does
+```
+
+This command is useful for:
+- Quick reference of available capabilities
+- Understanding tool signatures
+- Planning which tools to use for specific tasks
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)